src/main/java/com/google/sps/data/DatastoreEntityToObjectConverter.java

```diff
@@ -0,0 +1,83 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.Post;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory;
+
+/**
+ * Converts entities from datastore into objects 
+ */  
+public class DatastoreEntityToObjectConverter {
```

Comment:
since this class only has protected and private members, can we just make it protected, or even better package-private (what you get when there is no modified)? :)